contracts/src/bin/vendor.rs

```diff
@@ -30,6 +30,10 @@ fn run() -> Result<()> {
             "BalancerV2Vault",
             "balancer-labs/balancer-v2-monorepo/a3b570a2aa655d4c4941a67e3db6a06fbd72ef09/pkg/deployments/deployed/mainnet/Vault.json",
         )?
+        .github(
+            "BalancerV2WeightedPoolFactory",
```

Comment:
:+1: I think this is in the right spot as we would eventually need the bytecode for E2E test.